### ISSUE TITLE
Automatically add comment to pull request referencing a bugfix issue

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#32]
We're going to run into problems with people filing PRs against the 1.x branch of backdrop for bugfixes (which should go into point releases) instead of 1.0.x

If a PR references an issue that has the label of bug, we should add a comment to the PR as follows...
```
Thank you for helping us fix bugs in Backdrop!  Bug fixes go into point releases, so please make sure your PR is against a branch for a specific minor version (1.0.x) as opposed to the branch for the major version (1.x)
```

It would be even better if we could determine the branch the PR was filed against and close the PR immediately (or maybe mark it Needs Work) if it wasn't the correct one.

**Comment by sirkitree** [comment]
Has this been a problem yet, or another proactive idea?

**Comment by jenlampton** [comment]
Proactive idea - the more robots can do the less nate has to do :)

**Comment by quicksketch** [comment]
I've found that there's a manual element in this either way, since if I merge bug fixes into 1.0.x I have to merge 1.0.x into 1.x to get the change. If I do it the other way around, I merge bug fixes into 1.x and then I cherry-pick the fix back into 1.0.x. Neither takes any more effort than the other, so at this point I'm not sure it matters where people are filing their PRs.

**Comment by jenlampton** [comment]
It sounds like this wouldn't be a great help, so closing the issue.
